src/components/tools/pen.js

```diff
@@ -138,6 +139,22 @@ AFRAME.registerComponent("pen", {
     this.setDirty = this.setDirty.bind(this);
     this.dirty = true;
 
+    let material = new THREE.MeshStandardMaterial();
+    if (window.APP && window.APP.quality === "low") {
+      material = MobileStandardMaterial.fromStandardMaterial(material);
+    }
+    this.penTip = new THREE.Mesh(new THREE.SphereGeometry(1, 16, 12), material);
+    this.penTip.scale.setScalar(this.data.radius / this.el.parentEl.object3D.scale.x);
+    this.penTip.matrixNeedsUpdate = true;
+
+    this.el.setObject3D("mesh", this.penTip);
+
+    const environmentMapComponent = this.el.sceneEl.components["environment-map"];
+    if (environmentMapComponent) {
+      environmentMapComponent.applyEnvironmentMap(this.el.object3D);
+      environmentMapComponent.applyEnvironmentMap(this.el.parentEl.object3D);
```

Comment:
What is `parentEl` here? `applyEnvironmentMap` will traverse children anyway, so perhaps you can avoid the first call to it?

Reply:
the `parentEl` is the actual pen model- I didn't know that `applyEnvironmentMap` traverses so I can just call it on that.